5.5/mysql/bin/docker-entrypoint.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e
+
+# TODO: Only allow mysqld_safe?
+if [ ! -d '/var/lib/mysql/mysql' -a "${1%_safe}" = 'mysqld' ]; then
+	if [ -z "$MYSQL_ROOT_PASSWORD" ]; then
```

Comment:
It should print the usage if these vars are not set (instead of starting the mysql server with wrong parameters)